qlib/bytestream.rs

```diff
@@ -197,13 +197,23 @@ impl SocketBufIovs {
     }
 }
 
+#[cfg(not(feature = "cc"))]
 pub enum RingeBufAllocator {
     HeapAllocator,
     //ShareAllocator {headTailAddr: u64, bufAddr: u64},
     ShareAllocator(u64, u64, u64, bool),
 }
 
+#[cfg(feature = "cc")]
```

Comment:
there is `bytestream_cc.rs`, why modify the original then?

Reply:
Since there are repeated implementations.

Reply:
Yes, but in the original you have modifications like
```
fn xyz() {
    if cc {
        do abc
    } else {
        do otherwise
}
```
Then it's technically two functions. Why not have a clean implementation of xyz in both files?

Reply:
In btyestream.rs? you mean fn Init()?

Reply:
sorry I mean kernel.rs, mostly.

Reply:
I will try to fix the Kernel.rs part.